$OpenBSD: patch-src_libutil_util_c,v 1.1 2019/10/29 22:50:14 sthen Exp $

on OpenBSD we just have BLAS not OpenBLAS and this function doesn't exist.

Index: src/libutil/util.c
--- src/libutil/util.c.orig
+++ src/libutil/util.c
@@ -2502,7 +2502,7 @@ rspamd_config_libs (struct rspamd_external_libs_ctx *c
 			ZSTD_freeCStream (ctx->out_zstream);
 			ctx->out_zstream = NULL;
 		}
-#ifdef HAVE_CBLAS
+#if 0
 		openblas_set_num_threads (cfg->max_blas_threads);
 #endif
 	}
